# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/ClientWebGLExtensions.h
# Commit: 6e9a89ead3a5
# Full Hash: 6e9a89ead3a52864c48a4e5481e9a35e4aad5e98
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-11 04:04:33
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - WebXR compositing fixes. r=imanol
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78117
# ==============================================================================

diff -r dd00e2da3a0f -r 6e9a89ead3a5 dom/canvas/ClientWebGLExtensions.h
--- a/dom/canvas/ClientWebGLExtensions.h	Wed Jun 10 22:21:02 2020 +0000
+++ b/dom/canvas/ClientWebGLExtensions.h	Wed Jun 10 22:21:04 2020 +0000
@@ -317,7 +317,7 @@
       AutoJsWarning("present: Extension is `invalidated`.");
       return;
     }
-    mContext->Present();
+    mContext->OnBeforePaintTransaction();
   }
 };
 